spec/03-plugins/05-syslog/01-log_spec.lua

```diff
@@ -201,10 +202,57 @@ for _, strategy in helpers.each_strategy() do
 
         resp = stdout
       elseif expecting_same then
-        local _, _, stdout = assert(helpers.execute("find /var/log -type f -mmin -5 2>/dev/null | xargs grep -l " .. uuid))
-        assert.True(#stdout > 0)
-
-        resp = stdout
+        -- wait for file writing
+        helpers.pwait_until(function()
+          local _, _, stdout = assert(helpers.execute("sudo find /var/log -type f -mmin -5 | grep syslog"))
```

Comment:
is the file always going to be `/var/log/syslog` or if we are expecting other name?

Reply:
I'm not sure, the original logic was also to find suspicious files, so I just refined that logic.